app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
```

Comment:
better to use `()` then `\`
```suggestion
        return (
            f"{{Name: {self.name}, Health: {self.health}, " 
            f"Hidden: {self.hidden}}}"
        )
```